R/calcShares.R

```diff
@@ -99,6 +99,7 @@ calcShares <- function(subtype = c("carrier_nonthermal",
   # IEA ETP
   feETP <-
     calcOutput("ShareETP", subtype = shareOf, feOnly = TRUE, aggregate = FALSE) %>%
+    # toolCountryFillAvg() %>%
```

Comment:
This commented code could probably removed, I guess? (Might come from my commits, sorry!)